src/App.tsx

```diff
@@ -1,55 +1,100 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
```

Comment:
Fix it
<img width="876" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/596bcce5-20fb-4faa-9a9a-d621b5ed5789">